.github/CONTRIBUTING.md

```diff
@@ -1,50 +1,64 @@
-Contributing to Goss
-====================
+# Contributing to Goss
 
-Thank you for your interest in contributing to Goss. Goss wouldn't be where it is today if it wasn't for people like you. Some ways you can contribute:
+Thank you for your interest in contributing to Goss. Goss wouldn't be where it is today if it wasn't for people like you.
+Some ways you can contribute:
 
-* Improve the [README](https://github.com/goss-org/goss/blob/master/README.md) and/or [Docs](https://github.com/goss-org/goss/blob/master/docs/manual.md) - This makes it easier for new users to learn goss.
+* Improve the [README](https://github.com/goss-org/goss/blob/master/README.md)
+    and/or [Docs](https://github.com/goss-org/goss/blob/master/docs/).
+    This makes it easier for new users to learn goss.
 * Vote on bugs and feature requests by adding a :+1: reaction to the inital post.
 * Create tutorials, blog posts and example use-cases on how to use Goss.
 * Help users with [questions](https://github.com/goss-org/goss/labels/question) tracker.
 * Fix verified [bugs](https://github.com/goss-org/goss/issues?q=is%3Aopen+is%3Aissue+label%3Aapproved+label%3Abug+sort%3Areactions-%2B1-desc).
 * Implement approved [feature requests](https://github.com/goss-org/goss/issues?q=is%3Aopen+is%3Aissue+label%3Aapproved+label%3Aenhancement+sort%3Areactions-%2B1-desc).
 * Spread the word.
 
+## Features and bug reports and questions
```

Comment:
Yes he can, but to reduce the PR size and go iterative, I chose to only apply it on the documentation to start.
`.github` files are not part of it and just included when necessary as their primary purpose is to be recognized and used by github interface which is using Github-flavored Markdown.

I chose to have the minimal amount of change and new constraint on those file for the moment, to avoid spending time trying to enforce compatibility between 2 tools not using the same rules.

Reply:
Oh, yes; hadn't noted this was in `.github` 👍